tests/test_text/test_base.py

```diff
@@ -4,7 +4,7 @@
 # Author:   Benjamin Bengfort <bbengfort@districtdatalabs.com>
 # Created:  Mon Feb 20 06:34:50 2017 -0500
 #
-# Copyright (C) 2016 District Data Labs
+# Copyright (C) 2016 The scikit-yb developers
```

Comment:
I think this looks really good, I'm glad we're going this route! I think it also reinforces our Apache 2 license.